test/utils/operator/orderStatus.test.ts

```diff
@@ -113,65 +113,24 @@ describe('Filled status', () => {
   })
 })
 
-describe('Partially filled status', () => {
-  describe('Buy order', () => {
-    test('Partially filled, on the border to be considered filled', () => {
-      const order: RawOrder = { ...RAW_ORDER, kind: 'buy', buyAmount: '10000', executedBuyAmount: '9998' }
-
-      expect(getOrderStatus(order)).toEqual('partially filled')
-    })
-    test('Partially filled', () => {
-      const order: RawOrder = { ...RAW_ORDER, kind: 'buy', buyAmount: '10000', executedBuyAmount: '11' }
-
-      expect(getOrderStatus(order)).toEqual('partially filled')
-    })
-    test('Partially filled, sell amount does not affect output', () => {
-      const order: RawOrder = {
-        ...RAW_ORDER,
-        kind: 'buy',
-        buyAmount: '10000',
-        executedBuyAmount: '11',
-        sellAmount: '10000',
-        executedSellAmount: '123',
-      }
-
-      expect(getOrderStatus(order)).toEqual('partially filled')
-    })
+describe('Canceled status', () => {
+  test('Buy order', () => {
+    const order: RawOrder = {
+      ...RAW_ORDER,
+      kind: 'buy',
+      buyAmount: '10000',
+      invalidated: true,
```

Comment:
Should we make one test for testing that when EXPIRED + INVALIDATED ---> CANCELED